views/template/index.pug

```diff
@@ -0,0 +1,27 @@
+extends layout.pug
+
+block head
+	.container
+		.logo
+			img.head-logo(src="/assets/img/phpbblogo.png")
```

Comment:
I just kept it as a placeholder for the SDD logo. Do you want to keep this blank for now?